### DISCUSSION TITLE
!pip install -q git+https://github.com/pyannote/pyannote-audio is not working in google colab

### DISCUSSION THREAD

**Discussion by kaliNight** [openai/whisper#1750] [Q&A]
following by the below erorr
"WARNING: Ignoring invalid distribution -orch (/usr/local/lib/python3.10/dist-packages)
WARNING: Ignoring invalid distribution -orch (/usr/local/lib/python3.10/dist-packages)
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
openai-whisper 20230918 requires triton==2.0.0, but you have triton 2.1.0 which is incompatible."

I tried to install triton 2.0.0 but it still using trition 2.1.0

**Comment by hidenori-endo** [comment]
@kaliNight 
You may have to change pytorch version in Google colab.
torch above 2.1 seems to be depend on triton==2.1.0. so try torch-2.0.1 and torchaudio-2.0.2 :)
